silence/silencepb/silence.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package silencepb;
+
+import "google/protobuf/timestamp.proto";
+
+// Matcher specifies a rule, which can match or set of labels or not.
+message Matcher {
+  // Type specifies how the given name and pattern are matched
+  // against a label set.
+  enum Type {
+    EQUAL = 0;
```

Comment:
Yes, this was intentional in this case. I think its a sane default and having an `INVALID` would ultimately require validating a slice of matchers in all sorts of places.